spec/GraphQLOverHTTP.md

```diff
@@ -599,8 +599,8 @@ code `200` (Okay).
 
 ##### Document validation failure
 
-Requests that fail to pass _GraphQL validation_, the server SHOULD NOT execute
-the request and SHOULD return a status code of `200` (Okay).
+If a request fails to pass _GraphQL validation_, the server SHOULD NOT execute
+the request and SHOULD return a status code of `400` (Bad Request).
```

Comment:
I think this should be 200 for application/json responses

Reply:
You're right. Thanks for catching that. Fixed now.